linera-base/src/crypto/secp256k1/evm.rs

```diff
@@ -0,0 +1,635 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+// Copyright (c) Zefchain Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+//! Defines secp256k1 signature primitives used by the Linera protocol.
```

Comment:
```suggestion
//! Defines EVM-compatible secp256k1 signature primitives used by the Linera protocol.
```

(Or "Ethereum-compatible" or "MetaMask-compatible"?)

Reply:
It's EVM-wide standard but this is not entirely correct either. I should mention the EIP-191 here.